addons/binding/org.openhab.binding.netatmo/ESH-INF/thing/channels.xml

```diff
@@ -105,6 +123,33 @@
         <state readOnly="true"/>
     </channel-type>
     
+    <channel-type id="dateMinTemp" advanced="true">
+        <item-type>DateTime</item-type>
+        <label>Date Min Temp</label>
+        <description>Date when minimum temperature was reached</description>
+        <state readOnly="true"/>
+    </channel-type>
+    
+    <channel-type id="dateMaxTemp" advanced="true">
+        <item-type>DateTime</item-type>
+        <label>Date Max Temp</label>
+        <description>Date when maximum temperature was reached</description>
+        <state readOnly="true"/>
+    </channel-type>        
+    
+    <channel-type id="lastBilan" advanced="true">
+        <item-type>DateTime</item-type>
+        <label>Available Bilan</label>
+        <description>Month of the last available thermostat bilan</description>
+        <state readOnly="true"/>
+    </channel-type>
+    
+    <channel-type id="connectedBoiler" advanced="true">
+        <item-type>Switch</item-type>
+        <label>Plug Connected Boiler</label>
```

Comment:
This is not documented on Netatmo api, so I prefered not indicating a wrong description regarding this variable.